helm/ph-ee-kenya/config/application-bb.properties

```diff
@@ -0,0 +1,9 @@
+store.local.interop.host=https://rhino.mifos.g2pconnect.io
```

Comment:
lets use fynams.fynarfin.io

Reply:
Done.

Reply:
But shouldn’t it be sandbox.fynarfin.io instead?

Reply:
No. these are placeholder values. we will change them to the actual values in deployment specific charts.